test/e2e/reconciler/namespace/controller_test.go

```diff
@@ -90,12 +90,9 @@ func TestNamespaceScheduler(t *testing.T) {
 
 				t.Log("Deploy a syncer")
 				// Create and Start a syncer against a workload cluster so that there's a ready cluster to schedule to.
-				//
-				// TODO(marun) Extract the heartbeater out of the syncer for reuse in a test fixture. The namespace
-				// controller just needs ready clusters which can be accomplished without a syncer by having the
-				// heartbeater update the sync target so the heartbeat controller can set the cluster ready.
 				syncerFixture := framework.NewSyncerFixture(t, server, server.path,
-					framework.WithExtraResources("services")).Start(t)
+					framework.WithExtraResources("services"),
+				).Create(t).StartAPIImporter(t).StartHeartBeat(t)
```

Comment:
so "Deploy a syncer" is now wrong?

Reply:
Oh, just change the logs and comments, sure !

Reply:
Something like

`Create the SyncTarget and start both the Syncer APIImporter and Syncer HeartBeat`

would make sense to you ?

Reply:
Fixed in commit 13a4a065c80f0bf3815d69defc0ffcd0728fce2d